ludwig/utils/torch_utils.py

```diff
@@ -93,6 +93,23 @@ def losses(self):
             collected_losses.extend(layer.losses)
         return collected_losses
 
+
+class LudwigComponent(Module):
```

Comment:
We probably want to inherit from `LudwigModule` so we can get the loss tracking features.

Reply:
Done!